### PR TITLE
Update readme to recommend azure iot tools

### PR DESCRIPTION
**Pull request by adashen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
...
+- *Develop and connect your [Azure loT Applications](https://azure.microsoft.com/en-us/overview/iot/) to Azure. With this extension, you can interact with an Azure loT Hub, manage connected devices, and enable distributed tracing for your Azure loT applications.*
+- *Develop and debug [Certifies Azure loT Devices](https://catalog.azureiotsolutions.com/alldevices) (including [MXChip loT DevKit](http://www.mxchip.com/az3166), [ESP32](https://catalog.azureiotsolutions.com/details?title=ESP32_DevKitC&source=all-devices-page), [Raspberry Pi](https://www.adafruit.com/category/288)) to Azure. This extension pack makes it easy to code, build, deploy and debug your loT applications with popular loT development boards.*
+- *Develop and deploy artificial intelligence and your custom logic to (Azure loT Edge)[https://azure.microsoft.com/en-us/services/iot-edge/]. This extension pack makes it easy to code, build, deploy, and debug your loT Edge applications.*
```
Comment (formulahendry) [inline README.md]: Typo -> `(Azure loT Edge)[https://azure.microsoft.com/en-us/services/iot-edge/]`

Comment (adashen) [reply README.md]: without extension?

File: README.md [hunk 2]
```diff
@@ -1,6 +1,11 @@
 # Azure IoT Edge for Visual Studio Code
 [![Gitter](https://img.shields.io/badge/chat-on%20gitter-blue.svg)](https://gitter.im/Microsoft/vscode-azure-iot-edge)
 
+*[Azure IoT Edge extension](https://marketplace.visualstudio.com/items?itemName=vsciot-vscode.azure-iot-edge) is now part of [Azure IoT Tools](https://marketplace.visualstudio.com/items?itemName=vsciot-vscode.azure-iot-tools) extension pack. We highly recommend installing [Azure IoT Tools](https://marketplace.visualstudio.com/items?itemName=vsciot-vscode.azure-iot-tools) extension pack, which makes it easy to discover and interact with Azure IoT Hub that power your IoT Edge and device applications. This extension pack can help you:*
+- *Develop and connect your [Azure loT Applications](https://azure.microsoft.com/en-us/overview/iot/) to Azure. With this extension, you can interact with an Azure loT Hub, manage connected devices, and enable distributed tracing for your Azure loT applications.*
```
Comment (xinyiz1226) [inline README.md]: Azure loT Applications->Azure IoT Applications

Comment (xinyiz1226) [reply README.md]: looks like there is several typos using l(L) instead of I(i)

File: README.md (hunk 1 above)
Comment (formulahendry) [reply README.md]: The syntax of Markdown.  `()[]` should be `[]()`

Comment (adashen) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (adashen) [reply README.md]: fixed

**Pull request merged**